tchannel-core/pom.xml

```diff
@@ -138,6 +134,36 @@
             <groupId>junit</groupId>
             <artifactId>junit</artifactId>
         </dependency>
+        <dependency>
+            <groupId>io.netty</groupId>
+            <artifactId>netty-buffer</artifactId>
+            <version>4.1.11.Final</version>
```

Comment:
given we use `dependencyManagement` from parent pom, there should be no explicit artifact versions here. `<type>jar</type>` is also redundant.